lib/web_ui/dev/run.dart

```diff
@@ -33,11 +33,17 @@ class RunCommand extends Command<bool> with ArgUtils<bool> {
       help: 'Whether we require Skia Gold to be available or not. When this '
             'flag is true, the tests will fail if Skia Gold is not available.',
     );
+    argParser.addFlag(
+      'wasm',
+      help: 'Whether the test we are running are compiled to webassembly.'
+    );
   }
 
   @override
   String get name => 'run';
 
+  bool get wasm => boolArg('wasm');
```

Comment:
I think according to effective dart this should be `isWasm`: https://dart.dev/guides/language/effective-dart/design#prefer-a-non-imperative-verb-phrase-for-a-boolean-property-or-variable

Reply:
We should update this bool everywhere we pass it